lib/rautomation/adapter/helper.rb

```diff
@@ -19,6 +19,14 @@ def default_adapter
         end
       end
 
+      def valid_adapter?(adapter)
+        if Platform.is_x86?
+          [:win_32, :ms_uia].include?(adapter)
```

Comment:
What about `:autoit` and some unknown third party adapters, which might exist in the world? Simplify this logic to:
```
Platform.is_x86 || adapter == :win_32
```